apps/vs-code-designer/src/app/commands/generateDeploymentScripts/generateDeploymentScripts.ts

```diff
@@ -36,8 +39,24 @@ export async function generateDeploymentScripts(context: IActionContext, project
     ext.outputChannel.appendLog(localize('initScriptGen', 'Initiating script generation...'));
 
     addLocalFuncTelemetry(context);
-    const commandIdentifier = extensionCommand.generateDeploymentScripts;
-    showPreviewWarning(commandIdentifier);
+    showPreviewWarning(extensionCommand.generateDeploymentScripts);
+    const parameterizeConnectionsSetting = getGlobalSetting(parameterizeConnectionsInProjectLoadSetting);
+
+    if (!parameterizeConnectionsSetting) {
+      const message = localize(
```

Comment:
Hi @ecfan, could you please check the wording on this?

Reply:
I don't see any strings to review in the preceding highlighted section?

Reply:
Yes, that's what I meant. Sorry about the confusion